web/docs/tutorial/02-project-structure.md

```diff
@@ -15,7 +15,7 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
 
     After creating a new Wasp project and renaming the `src/MainPage.tsx` file, your project should look like this:
 
-    ```
+    ```python
```

Comment:
🐍

Reply:
can we have a internal comment that this is only python to improve syntax highlighting?